Modules/xASL_module_ASL.m

```diff
@@ -41,6 +41,7 @@
 % - D3. Multi-PLD parsing
 % - D4. TimeEncoded parsing
 % - D5. Multi-TE parsing
+% - D6. DeltaM parsing - check if all/some volumes are deltams
```

Comment:
Nice, but then this should include that if we have a single volume only, we assume it is a `deltaM`

Reply:
Good point!!! Will do that - see comments above.